website/project/views/register.py

```diff
@@ -254,3 +257,18 @@ def get_referent_by_identifier(category, value):
     if identifier.referent.url:
         return redirect(identifier.referent.url)
     raise HTTPError(http.NOT_FOUND)
+
+@must_be_signed
+@must_be_registration
+def registration_callbacks(node, payload, *args, **kwargs):
```

Comment:
This may be elsewhere in the code, but should this be creating logs?
That way users can see the exact time that a certain addon finished archiving.